ambry-messageformat/src/main/java/com/github/ambry/messageformat/BlobPropertiesSerDe.java

```diff
@@ -27,29 +27,35 @@
  */
 public class BlobPropertiesSerDe {
 
-  static final short VERSION_1 = 1;
-  static final short VERSION_2 = 2;
-  static final short VERSION_3 = 3;
-  static final short VERSION_4 = 4;
+  public static final short VERSION_1 = 1;
+  public static final short VERSION_2 = 2;
+  public static final short VERSION_3 = 3;
+  public static final short VERSION_4 = 4;
+  public static final short VERSION_5 = 5;
   private static final int VERSION_FIELD_SIZE_IN_BYTES = Short.BYTES;
   private static final int TTL_FIELD_SIZE_IN_BYTES = Long.BYTES;
   private static final int PRIVATE_FIELD_SIZE_IN_BYTES = Byte.BYTES;
   private static final int CREATION_TIME_FIELD_SIZE_IN_BYTES = Long.BYTES;
   private static final int BLOB_SIZE_FIELD_SIZE_IN_BYTES = Long.BYTES;
   private static final int ENCRYPTED_FIELD_SIZE_IN_BYTES = Byte.BYTES;
+  public static short CURRENT_VERSION = VERSION_4;
```

Comment:
In the BlobPropertiesSerDe, there are some comments referring to #VERSION_4 directly. Do we want to change it?
```
   * Serialize {@link BlobProperties} to buffer in the {@link #VERSION_4}
   *
```

Reply:
fixed it.